test/Microsoft.NET.TestFramework/TestContext.cs

```diff
@@ -145,7 +139,7 @@ public static void Initialize(TestCommandLine commandLine)
             {
                 testContext.TestExecutionDirectory = (Path.Combine(FindFolderInTree("artifacts", AppContext.BaseDirectory), "tmp", repoConfiguration));
 
-                testContext.TestAssetsDirectory = FindFolderInTree(Path.Combine("src", "Assets"), AppContext.BaseDirectory);
+                testContext.TestAssetsDirectory = FindFolderInTree(Path.Combine("test", "TestAssets"), AppContext.BaseDirectory);
```

Comment:
Can we put `test` and `TestAssets` in constants somewhere. They appear more than once and someone is going to miss making a future change and only modify one occurrence.

Reply:
I do agree with @joeloff it's a wonderful idea to have these as constants somewhere, but I also think that's a big enough change that it would be easier to review in a separate PR as @MiYanni suggests.